test/views/propertyDetails/PropertyDetailsRevaluedViewSpec.scala

```diff
@@ -153,7 +151,6 @@ feature("The user can view an empty property revalue page") {
         assert(document.title() === "Has the property been revalued since the £40,000 or more change? - GOV.UK")
 
         And("The data is populated for a property value set to false")
-        assert(document.getElementById("isPropertyRevalued").text() === "Has the property been revalued since the £40,000 or more change? Yes No")
```

Comment:
Can you change the test rather than remove it. We need to test that something is there.